### PR TITLE
when a number of tuning-phases is specified finish the simulation whe…

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

MD-Flex can now be run until a specified number of tuning phases is completed

## Related Pull Requests

- merge after #454 (merge target will be master. Currently points to super-class.... for reviewing reasons)

## Resolved Issues

# How Has This Been Tested?

- [x] playing around with md-flex

### CODE REVIEW

**Comment by FG-TUM** [comment]
@kruegener : this feature will probably become interesting for you for your upcoming performance measurement script.

@julianpelloth : same

File: examples/md-flexible/Simulation.h [hunk 1]
```diff
@@ -235,6 +257,11 @@ void Simulation<Particle, ParticleCell>::initialize(const MDFlexConfig &mdFlexCo
     Checkpoint::loadParticles(_autopas, _config->checkpointfile);
   }
 
+  // sanitize simulation end condition
+  if (_config->tuningPhases > 0) {
+    _config->iterations = std::numeric_limits<size_t>::min();
```
Comment (SteffenSeckler) [inline examples/md-flexible/Simulation.h:262]: so 0?

Comment (SteffenSeckler) [reply examples/md-flexible/Simulation.h:262]: you want max here, don't you?

Comment (SteffenSeckler) [reply examples/md-flexible/Simulation.h:262]: actually was ok. But maybe use `0ul` here

**Comment by FG-TUM** [comment]
@SabrinaKrallmann  this is the option I told you about

**Pull request merged**